.github/workflows/auto-review-comment.yml

```diff
@@ -0,0 +1,34 @@
+name: Auto Review Comment
+
+on:
+  workflow_run:
+    workflows:
+      - Diff Compile
+    types:
+      - completed
+
+permissions:
+  pull-requests: write
+
+jobs:
+  auto-review:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Download Diff Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: auto-review-artifacts
+          path: auto_review_artifacts/
+          repository: ${{ github.repository_owner }}/${{ github.event.repository.name }}
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are you sure that it will work with standard `GITHUB_TOKEN` and we do not need special token?